2022/globalazuremumbai/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "indcoder",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Global Azure Bootcamp 2022 Mumbai",
+    "Country": "India",
+    "Url": "https://hackmum.in/",
+    "SessionizeApiId": "https://sessionize.com/gab22mum/",
```

Comment:
This is not the event sessionizeid, right @davidjrh!

Reply:
Yes, my bad. I had inadvertently put the CfP URl. Apologies

Reply:
@noopman SessionizeAPI  has been updated with the API code

Reply:
@indcoder @noopman yes, now it is fine. Merged!